resources/public/api/conf/1.0/application.raml

```diff
@@ -38,6 +38,12 @@ uses:
         description: National Insurance number, in the format AA999999A.
         type: string
         example: TC663795B
+      bsasId:
```

Comment:
Can this be moved to below line 45? So the definition of `bsasId` is directly below where it's used rather than above it. ie:

```
    /self-employment/{bsasId}:
      uriParameters:
        bsasId:
          description: 'The unique identifier of the summary calculation.<br><br>Must conform to the regular expression: `^[0-9]{8}|[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$`'
          type: string
          example: f2fb30e5-4ab6-4a29-b3c1-c7264259ff1c
      get: !include retrieve_se_bsas.raml
```